src/brevitas/core/quant/float.py

```diff
@@ -86,6 +88,13 @@ def dequantize(self, y, scale):
     @brevitas.jit.script_method
     def forward(self, x):
         y, scale = self.quantize(x)
+        # after quantizing, clamp to special cases like NaN/inf if they are set
+        if self.case_clamp_impl is not None:
+            y = self.case_clamp_impl(
+                y,
+                exponent_bit_width=self.exponent_bit_width(),
+                mantissa_bit_width=self.mantissa_bit_width(),
+                exponent_bias=self.exponent_bias())
```

Comment:
Put the computation of the exponent_bias in the quantizer, and it can never be None.
Then you can build case_clamp_impl from dep_inj with all these attributes that are not defined at runtime